src/compiler/types.ts

```diff
@@ -790,6 +791,10 @@ module ts {
 
     export interface SwitchStatement extends Statement {
         expression: Expression;
+        caseBlock: CaseBlock;
+    }
+
+    export interface CaseBlock extends Node {

```

Comment:
Can we just add a comment that this is more like a SwitchBlock? It's somewhat misleading